app/event-list/event-list.component.html

```diff
@@ -14,31 +14,11 @@
                 <StackLayout row="1" col="0" backgroundColor="#e0e0e0" class="list-group-item">
                     <Label class="list-group-item-heading" [text]="event.name"></Label>
                     <Label class="list-group-item-text fa" text="&#xf073;" [text]="'Start Date: July 22, 2018'"></Label>
-                    <Label class="list-group-item-text fa" text="&#xf073;" [text]="'End Date: July 28, 2018'"></Label>          
+                    <Label class="list-group-item-text fa" text="&#xf073;" [text]="'End Date: July 28 2018'"></Label>    
+                    <!-- <Label class="list-group-item-text fa" text="&#xf073;" [text]="event.address + event.endDate"></Label>     -->
```

Comment:
Since this isn't done yet, can we track this in the excel sheet as a "high" item work?

Reply:
Added